p13.ts

```diff
@@ -0,0 +1,11 @@
+import "date-wizard";
```

Comment:
The solution is correct although the rest of the code is missing in order to run it.